### PR TITLE
feat: Created confirmation for elevating user to admin

### PR DESCRIPTION
**Pull request by Althaf66**
@amands98 ,

Completed the TODO in the elevate command.

Created confirmation for elevating user to admin.

### CODE REVIEW

File: cmd/harbor/root/user/elevate.go [hunk 1]
```diff
@@ -28,14 +29,22 @@ func ElevateUserCmd() *cobra.Command {
...
 			}
 
-			// Todo : Ask for the confirmation before elevating the user to admin role
+			var confirm bool
 
-			err = runElevateUser(userId)
-
-			if err != nil {
-				log.Errorf("failed to elevate user: %v", err)
+			huh.NewConfirm().
```
Comment (amands98) [inline cmd/harbor/root/user/elevate.go:34]:
```go
func confirmElevation() (bool, error) {
	var confirm bool
	|
	|
	|
	.
	.
	.

	return confirm, nil
}
```


Create a separate confirmation model in the 'views' directory and name the file 'confirmation.go'.

Comment (Althaf66) [reply cmd/harbor/root/user/elevate.go]: Yes. I created confirmation.go file.
Please review it @amands98

**Review by amands98 — COMMENTED** [review]
please run lint the file using `make` command or `gofmt -s -w .`

**Comment by Althaf66** [comment]
make changes,
ready for review @amands98

**Review by amands98 — APPROVED** [review]
LGTM :rocket:

**Review by Vad1mo — APPROVED** [review]

**Pull request merged**